build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="test/ProjectsToTest.props" />
+  <Import Project="test/TestPackageProjects.props" />
+
+  <Target Name="PrepareTests"
+          DependsOnTargets="Prepare">
```

Comment:
How should I think about Init vs. Prepare? Can we improve the naming to better express the intent? I'm happy to make the change, but it's not clear where we want to be headed.